src/Tizen.NUI/src/public/Events/TapGesture.cs

```diff
@@ -87,6 +87,20 @@ public Vector2 LocalPoint
             }
         }
 
+        /// <summary>
+        /// The gesture source type of touches property (read-only).
+        /// If you tap with a mouse button, this will tell you which mouse input you tapped.
+        /// Primary(Left), Secondary(Right). Center(Wheel).
```

Comment:
It is not matched with `GestureSourceType` you intent `Tertiary`?
```suggestion
        /// Primary(Left), Secondary(Right). Tertiary(Wheel).
```